openassessment/assessment/test/test_peer.py

```diff
@@ -1253,11 +1251,11 @@ def test_median_score_db_error(self, mock_filter):
         tim, _ = self._create_student_and_submission("Tim", "Tim's answer")
         peer_api.get_assessment_median_scores(tim["uuid"])
 
-    @patch.object(PeerWorkflowItem, 'get_scored_assessments')
+    @patch.object(Assessment.objects, 'filter')
     @raises(peer_api.PeerAssessmentInternalError)
     def test_get_assessments_db_error(self, mock_filter):
-        mock_filter.side_effect = DatabaseError("Bad things happened")
         tim, _ = self._create_student_and_submission("Tim", "Tim's answer")
+        mock_filter.side_effect = DatabaseError("Bad things happened")
```

Comment:
`side_effect` has to be defined here, otherwise it will blow up in the above line instead of the one we want.